lms/envs/common.py

```diff
@@ -2213,6 +2213,11 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 # parallel, and what the SES rate is.
 BULK_EMAIL_RETRY_DELAY_BETWEEN_SENDS = 0.02
 
+# Setting that can be configured in order for those interested in just using
+# the default configured email address as the from_addr to save them from
+# dealing with issues with spam filtering
+EMAIL_USE_DEFAULT_FROM_FOR_BULK = False
```

Comment:
This new toggle needs to be annotated according to [OEP-17](https://open-edx-proposals.readthedocs.io/en/latest/oep-0017-bp-feature-toggles.html). Here's a suggestion:

```
# .. toggle_name: EMAIL_USE_DEFAULT_FROM_FOR_BULK
# .. toggle_implementation: DjangoSetting
# .. toggle_default: False
# .. toggle_description: If True, use DEFAULT_FROM_EMAIL as the from_addr for bulk mail, to avoid issues with spam filtering
# .. toggle_use_cases: open_edx
# .. toggle_creation_date: 2020-09-14
# .. toggle_tickets: OSPR-4957 
```

I think the description is accurate, but you should definitely review that.